hydradx/model/processing.py

```diff
@@ -908,3 +920,291 @@ def find_data_files():
                     tkn_data += [line for line in json.load(f) if line[2] == tkn]
         print(f"Retrieved balance of {tkn} from {date.strftime('%Y-%m-%d')} - {end_date.strftime('%Y-%m-%d')}.")
         return {data[1]: data[3] for data in tkn_data[left:end]}
+
+
+def distribute_value(num_positions, total_value, concentration = 0):
```

Comment:
Is this being used? Is it being tested? I'm trying to figure out what this function is supposed to do...

Reply:
It's not being used. I had the idea early on to try this as a way of distributing value in the CDPs (based on something Kubo said at one point) but I ended up going with the downloaded data instead.

Reply:
I think it's a cool function, but it's not being used so it doesn't have to stay.